go.mod

```diff
@@ -19,6 +19,7 @@ require (
 	k8s.io/apiextensions-apiserver v0.21.0-beta.1
 	k8s.io/apimachinery v0.21.0-beta.1
 	k8s.io/client-go v0.21.0-beta.1
+	k8s.io/cluster-bootstrap v0.21.0-beta.1
```

Comment:
hmmm I'd expect something like api-machinery to have utility func to get data, might be worthwhile comparing with other providers

Reply:
CAPA seems to do it themselves. It's a small enough util so just added it to the identity package as well